src/main/java/frc/robot/subsystems/DriveBase.java

```diff
@@ -20,14 +20,12 @@
 public class DriveBase extends SubsystemBase {
 
     // DriveBase is a singleton class as it represents a physical subsystem
-    private static DriveBase currentInstance;
+    private static DriveBase INSTANCE = new DriveBase();
```

Comment:
Can we revert this to the old Singleton Pattern? Or better yet this pattern: https://github.com/georgetzavelas/guava-examples/blob/master/src/com/design/pattern/creational/singleton/Singleton.java